boards/riscv/it8xxx2_evb/it8xxx2_evb.yaml

```diff
@@ -16,3 +16,4 @@ supported:
   - kscan
   - pwm
   - watchdog
+vendor: riscv
```

Comment:
@Dino-Li @GTLin08 @RuibinChang looks like this one and `it82xx2_evb` boards have wrong vendor set in the devicetree. Could you take a look at this?

Reply:
@nashif `ite` is correct. Created PR to fix dts #59738